src/app/shared/dialogs/dialogs-images.component.ts

```diff
@@ -48,6 +48,12 @@ export class DialogsImagesComponent implements OnInit {
 
   uploadImage(event) {
     const file = event.target.files[0];
+    const imageExists = this.images.filter( img => file.name === img.filename).length > 0;
+    if (imageExists) {
+      this.planetMessageService.showAlert(`An image with that  ${file.filename}
+        filename exists. Please rename or select other image.`);
```

Comment:
Please put the string on one line.  When using backticks the newlines in the file are added to the string.